AWS/aws-cloudtrail/ingest/parser.yml

```diff
@@ -48,22 +48,28 @@ stages:
           user.name: '{{json_event.message.responseElements.user.userId}}'
           tls.cipher: '{{json_event.message.tlsDetails.cipherSuite}}'
           tls.version: '{{json_event.message.tlsDetails.tlsVersion}}'
+          source.domain: '{{parsed_source.source.domain}}'
+
       - set:
           source.address: '{{json_event.message.sourceIPAddress}}'
+        filter: '{{json_event.message.sourceIPAddress | is_ipaddress}}'
+
+      - set:
           source.ip: '{{parsed_source.source.ip}}'
-          source.domain: '{{parsed_source.source.domain}}'
-        filter: '{{json_event.message.sourceIPAddress != "AWS Internal"}}'
```

Comment:
Is this filter obsolete ?

Reply:
I reintroduced this filter thank you :smile_cat: